src/Pages/TagSearch/Components/Tags.js

```diff
@@ -0,0 +1,50 @@
+import React, { useState, useEffect } from "react";
+import styled from "styled-components";
+import Tag from "./Tag";
+
+function Tags() {
+  const [tagData, setTagData] = useState([]);
+
+  const getTagData = async () => {
+    const res = await fetch(`http://192.168.0.23:8000/companies/recommend`);
+    const getDataList = await res.json();
+    setTagData([getDataList.tags]);
+  };
+
+  useEffect(() => {
+    getTagData();
+  }, []);
+  console.log(tagData);
+  return (
+    <TagsContainer>
+      <TagMenu>
+        <Reco>추천</Reco>
+        {tagData.length &&
+          tagData[0].map((el) => <Tag key={el.tag_id} tagName={el.tag_name} />)}
```

Comment:
```suggestion
        {tagData[0]?.map((el) => <Tag key={el.tag_id} tagName={el.tag_name} />)}
```
옵셔널 체이닝 써주면 이렇게 써줄 수 있겠네요

Reply:
넵 감사합니다!